backend/dal/Pims.Dal.csproj

```diff
@@ -36,6 +36,8 @@
     <PackageReference Include="ProjNet" Version="2.0.0" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.2" />
     <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
+    <PackageReference Include="Macross.Json.Extensions" Version="2.2.0" />
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
Needed to add these to be able to use `JsonConverter(typeof(JsonStringEnumMemberConverter))` in the enum - basically supporting string enums in JSON